packages/admin-stories/src/admin/toolbar/samples/ToolbarAutomaticTitle.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+    MainContent,
+    Stack,
+    StackPage,
+    StackSwitch,
+    StackSwitchApiContext,
+    Toolbar,
+    ToolbarActions,
+    ToolbarBackButton,
+    ToolbarFillSpace,
+    ToolbarTitleItem,
+} from "@comet/admin";
+import { Box, Button, Container, Typography } from "@material-ui/core";
+import * as React from "react";
+
+export const ToolbarAutomaticTitle = () => {
+    return (
+        <Stack topLevelTitle={"Automatic Title"} showBreadcrumbs={false} showBackButton={false}>
+            <StackSwitch initialPage="automaticTitle">
+                <StackPage name={"automaticTitle"}>
+                    <>
+                        <Toolbar>
+                            <ToolbarBackButton />
+                            <ToolbarTitleItem />
```

Comment:
I'm not sure this Automatic Title feature is helpful: instead of hardcoding the title at `<Stack topLevelTitle={"Automatic Title"}` and `<StackPage title={"Automatic Title Detail"}>` we could hardcode it right here. (yes, topLevelTitle is required, but we could change that)

Pro "Automatic Title": 
topLevelTitle and title are required though when using `ToolbarBreadcrumbs` (or the "old" breadcrumbs), so switching between `ToolbarTitleItem` and `ToolbarBreadcrumbs` would be very easy.

Con "Automatic Title":
In most cases we won't use `ToolbarBreadcrumbs`, no need for this complicated (compared to hardcoding) mechanism

Reply:
i am not sure either.

- i like the idea of setting the title just in one place (either on Stack.topLevelTitle or on a StackPage.title), reusing it in another way (`ToolbarTitleItem`) and having the possibility to customise it in another way (`CustomToolbarTitle`).
- why would i use a  different title on StackPage and the Toolbar?  I think mostly this should be the same title.
- Furthermore some things get generated based on this titles (e.g. Breadcrumbs, maybe other?) and this depend on this title's.
- I think it could be nice for upgrading older projects, where title on the Stack Pages and the topLevelTitle is already set. Simple drop in this component and the projects has the correct title. Migrating from the old Breadcrumbs / Back Button could be easier by drop in's



Furthermore i think it is such a small component (sometimes helpfull component - sometimes not), if someone does not want to use it, he is not forced to use it.


**Further thoughts:**
I also thought about adding one more layer for consistensy over the whole application: ( but i am not sure if we need this) where the Typography is set automatically by the theme.

```
<ToolbarTitleItem>
Lorem Ipsum
</ToolbarTitleItem>
```

```
<ToolbarAutomaticTitleItem/>
```

Reply:
> if someone does not want to use it, he is not forced to use it.

absolutely true. Question remains, if we use Automatic Title as best pratice (starter, docs except from Toolbar doc itself) or not.

> I also thought about adding one more layer for consistensy over the whole application

to avoid `<Typography variant={"h3"}>`? yes, good idea.

Reply:
just added `ToolbarAutomaticTitleItem` and `ToolbarTitleItem`